tests/e2e/project.spec.ts

```diff
@@ -15,11 +16,13 @@ async function expectCounts(
   page: Page,
 ) {
   await expect(page.locator('role=button[name="Commit count"]')).toContainText(
-    `${params.commits} commit(s)`,
+    `${params.commits} ${pluralize("commit", params.commits)}`,
```

Comment:
We should just hardcode the expectation. Otherwise our test result depends on the implementation of this function and that is now what is under test, so we're mixing concerns.